packages/launchpad/src/layouts/Layout.vue

```diff
@@ -70,27 +72,77 @@ query Layout {
       title
     }
   }
+
+  navigationMenu {
+    items {
+      id,
+      name
+      selected
+      iconPath
+    }
+  }
 }
 `
 
+gql`
+mutation NavigationMenuSetItem($type: NavItem!) {
+  navigationMenuSetItem (type: $type) {
+    items {
+      name
+      selected
+    }
+  }
+} 
+`
+
+const icons = {
```

Comment:
not super thrilled about this coupling but not really sure how else to handle this 🤔

Reply:
This an unfortunate consequence of handling the front-end state on the back-end.
The only way out of this that I see is to keep the navigation out of GQL.

I think this is handled right